auto_submit/lib/model/auto_submit_query_result.dart

```diff
@@ -178,3 +180,35 @@ class QueryResult {
 
   Map<String, dynamic> toJson() => _$QueryResultToJson(this);
 }
+
+@JsonSerializable()
+class OuterRevertPullRequest {
```

Comment:
Is this class necessary? I am not sure how this class help the logics.

Reply:
Yes. In general it helps the functions that run the query and mutations from being too specific. By having this outer class it allows the decoding of the json to be simpler rather than having to specify and extra step after calling the function to get this from the json map then decode. The decoding happens for us when this is specified.